test/browser/browser.sh

```diff
@@ -83,10 +51,16 @@ if [ "$ID" = fedora ]; then
     dnf install -y virtiofsd
 fi
 
-# Run tests as unprivileged user
-# once we drop support for RHEL 8, use this:
-# runuser -u runtest --whitelist-environment=TEST_BROWSER,TEST_ALLOW_JOURNAL_MESSAGES,TEST_AUDIT_NO_SELINUX,SOURCE,LOGS $TESTS/run-test.sh $PLAN
-runuser -u runtest --preserve-environment env USER=runtest HOME=$(getent passwd runtest | cut -f6 -d:) $TESTS/run-test.sh $PLAN
-
-RC=$(cat $LOGS/exitcode)
-exit ${RC:-1}
+# Run tests in the cockpit tasks container, as unprivileged user
+CONTAINER="$(cat .cockpit-ci/container)"
+exec podman \
+    run \
+        --rm \
+        --shm-size=1024m \
+        --security-opt=label=disable \
+        --network=host \
```

Comment:
Do you want to make `--network=host` the "standard way" of doing this, or will we keep this as an adaption for cockpit-machines only?

In that case, I think a comment might be helpful here to explain why we deviate from starter-kit.

Reply:
I like the net ns separation by default. I'll add a comment. I just wanted to get an agreement with you before I push again.

Reply:
Can you see my comment in the starter-kit PR as well about `/run/host` and do that?

Reply:
Done.

Reply:
Changed the bind mount path to /run/host